specification/sql/resource-manager/Microsoft.Sql/preview/2021-08-01-preview/IPv6FirewallRules.json

```diff
@@ -0,0 +1,377 @@
+          },
+          "204": {
+            "description": "The specified IPv6 firewall rule does not exist."
+          }
+        },
+        "x-ms-examples": {
+          "Delete an IPv6 firewall rule": {
+            "$ref": "./examples/IPv6FirewallRuleDelete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/ipv6FirewallRules": {
+      "get": {
+        "tags": [
+          "IPv6FirewallRules"
+        ],
+        "description": "Gets a list of IPv6 firewall rules.",
+        "operationId": "IPv6FirewallRules_ListByServer",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of IPv6 firewall rules.",
+            "schema": {
+              "$ref": "#/definitions/IPv6FirewallRuleListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List IPv6 Firewall Rules": {
+            "$ref": "./examples/IPv6FirewallRuleList.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
PUT is only for a single resource creation or update. Collection PUT is a RPC violation.
https://github.com/Azure/azure-resource-manager-rpc/blob/d9c0275c7ada1a65f25ec85d88b3f8fe0bf3dc1e/v1.0/resource-api-reference.md#put-resource

Reply:
waiting for the decision from the feature owner.

Reply:
to unblock the release, we remove this batch PUT API from this version in commit: c930be5c58c9b8d759b272eafe72229b30061fe6
will add it to the next version by the feature owner.